enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/log/LogPosition.java

```diff
@@ -29,4 +31,22 @@ public LogPosition( long logIndex, long byteOffset )
         this.logIndex = logIndex;
         this.byteOffset = byteOffset;
     }
+
+    @Override
+    public boolean equals( Object o )
+    {
+        if ( this == o )
+        { return true; }
+        if ( o == null || getClass() != o.getClass() )
+        { return false; }
```

Comment:
formatting is wrong here, strange that this didn't upset the checkstyle guardian...